.codeclimate.yml

```diff
@@ -0,0 +1,15 @@
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+        javascript:
+          mass_threshold: 100
+ratings:
+  paths:
+  - src/js/**
+exclude_paths:
```

Comment:
I think the exclude paths remove files from any of the `path` options used (there's only one in here, but there can be more depending on the engines you use). Might not be useful in our case, actually.